app/assets/stylesheets/frontend/base.scss

```diff
@@ -142,7 +142,8 @@ $govuk-typography-use-rem: false;
 // When removing ensure all cases are captured - some rules in helpers/_headings.scss
 // are very specific, requiring the !important here
 #whitehall-wrapper {
-  a.govuk-link{
+  a.govuk-link,
+  a[class^="gem-c-"]{
```

Comment:
Sorry, I've probably given this a conflict here as I added it in as well but with a space between the `]` and the `{`.